inspec.yml

```diff
@@ -0,0 +1,370 @@
+name: cms-ars-5.0-microsoft-windows-server-2019-stig-overlay
+title: cms-ars-5.0-microsoft-windows-server-2019-stig-overlay
+maintainer: CMS InSpec Dev team
+copyright:
+copyright_email:
+license: Apache-2.0
+summary: "CMS ARS 5.0 Overlay InSpec Validation Profile for Windows Server 2019 STIG"
+version: 1.3.0
+inspec_version: ">= 4.0"
+
+depends:
+- name: microsoft-windows-server-2019-stig-baseline
+  url: https://github.com/mitre/microsoft-windows-server-2019-stig-baseline/archive/parameterize.tar.gz
```

Comment:
@ejaronne The branch name is currently set to `parameterize` for testing purposes. It would need to be changed to `master` before the PR is merged.

Reply:
Switched to `master`